actors/miner/tests/util.rs

```diff
@@ -204,8 +261,1023 @@ impl ActorHarness {
         let value = result.deserialize::<GetControlAddressesReturn>().unwrap();
         (value.owner, value.worker, value.control_addresses)
     }
+
+    pub fn commit_and_prove_sectors(
+        &mut self,
+        rt: &mut MockRuntime,
+        num_sectors: usize,
+        lifetime_periods: u64,
+        deal_ids: Vec<DealID>,
```

Comment:
This mismatches go actors in a way that will need to be fixed later for ease of use.  It should be `Vec<Vec<DealID>>`, each sector can have multiple deals.  For this PR there are no deals so no problem leaving it as long as you leave a follow up issue.

Reply:
added TODO.